apollo-compiler/src/test/graphql/com/example/fragment_in_fragment/TestOperation.graphql

```diff
@@ -14,7 +14,7 @@ fragment starshipFragment on Starship {
   pilotConnection {
     edges {
       node {
-        ...pilotFragment
+       name
```

Comment:
Is it intentionally? As this test was used for testing fragment inside another fragment